cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/CruiseControlMetricsReporterSampler.java

```diff
@@ -90,11 +90,11 @@ public Samples getSamples(Cluster cluster,
     }
     LOG.debug("Starting consuming from metrics reporter topic.");
     _metricConsumer.resume(_metricConsumer.paused());
-    int numMetricsAdded;
     int totalMetricsAdded = 0;
     long maxTimeStamp = -1L;
+    // Ideally we should have a timeout passed in to the metric sampler. Will do that in a separate patch.
```

Comment:
Can we add a `TODO:` keyword before the comment so that we can find & fix this comment easily in the future.